### ISSUE TITLE
[bug] android invisibility rapidly switching on/off

### ISSUE THREAD

**Issue by shadoxxhd** [simeonradivoev/MatterOverdrive#366]
i keep reappearing every 0.5-2s for a split secons, and i always get to hear the cloak on/off noise. right after installation the cloaking ability worked flawlessly and kinda 'broke' some time later. any idea what could cause this (also happens without armor equipped/item in hand)?

**Comment by shobu9** [comment]
TBH, i think cloak still needs work lol.

**Comment by Jacob345k** [comment]
No, this is because most likely you have an advanced genetics gene, at least in my testing it was because of this.

**Comment by shadoxxhd** [comment]
I did not have advanced genetics installed at that point. the hoanna
radiation somehow rapidly decloaked me - maybe it dealt 0-damage-ticks for
some effect?
